config/provisioners/in-memory-channel/in-memory-channel.yaml

```diff
@@ -92,6 +92,17 @@ rules:
       - watch
       - create
       - update
+  - apiGroups:
+      - "" # Core API Group.
+    resources:
+      - events
+    verbs:
+      - get
```

Comment:
Just wondering if we need all the verbs here or just 'create'?

Reply:
Create is not enough. I tried patching a running cluster and didn't work.
I can remove things out and leave the ones necessary. Might just be create and patch...
Let me double check